src/steamship/data/package_version.py

```diff
@@ -50,12 +50,12 @@ def create(
                 filebytes = f.read()
 
         req = CreatePackageVersionRequest(
-            handle=handle, app_id=app_id, config_template=config_template
+            handle=handle, package_id=package_id, config_template=config_template
         )
 
         return client.post(
-            "app/version/create",
+            "package/version/create",
             payload=req,
-            file=("app.zip", filebytes, "multipart/form-data"),
+            file=("invocable.zip", filebytes, "multipart/form-data"),
```

Comment:
Shouldn't this be package.zip? 

```suggestion
            file=("package.zip", filebytes, "multipart/form-data"),
```